spot-webdriver/specs/spot-tv-reconnect.spec.js

```diff
@@ -3,6 +3,24 @@ const spotSessionStore = require('../user/spotSessionStore');
 describe('The reconnect logic', () => {
     const session = spotSessionStore.createSession();
 
+    /**
+     * Helper to connect the Spot-TV to the Spot-Remote and join a meeting.
+     *
+     * @private
+     * @returns {void}
+     */
+    function joinMeeting() {
+        const tv = session.getSpotTV();
+        const remote = session.getSpotRemote();
+
+        session.connectRemoteToTV();
+
+        session.joinMeeting();
+
+        remote.getInMeetingPage().waitForVisible();
+        tv.getMeetingPage().waitForMeetingJoined();
```

Comment:
Why not move those waits to `session.joinMeeting` and remove this method? I just feel like this logic will be duplicated among other tests.

Reply:
I've moved any waits I found after joinMeeting() into joinMeeting().

Reply:
I take that back. that introduced failing tests. I'll get back to this change.